src/utils/utils/utils.ts

```diff
@@ -157,3 +159,17 @@ export const removeDockerPrefix = (image: string) => image?.replace(DOCKER_PREFI
 
 export const isAllNamespaces = (namespace: string) =>
   !namespace || namespace === ALL_NAMESPACES || namespace === ALL_NAMESPACES_SESSION_KEY;
+
+export const isClusterOnS390X = !isEmpty(
+  (window as any).SERVER_FLAGS?.nodeArchitectures?.find((arch) => arch === S390X_ARCH),
```

Comment:
It's worth noting that this isn't a public API and could change. I wouldn't anticipate it changing, but it could. I'd suggest opening a management console issue to provide a supported API (and you can fallback to `SERVER_FLAGS` for older console versions).

@jhadvig @TheRealJon FYI, the kubevirt plugin is using `window.SERVER_FLAGS.nodeArchitectures`, so we should avoid removing or changing that if possible.

Reply:
Wouldn't this issue apply to any SERVER_FLAG value? These are just available globally at runtime and could be used by any plugin, so making breaking changes to them is not very safe.

Reply:
Yes. `SERVER_FLAGS` are internal APIs. Any use by plugins could break in a future version.

We mention that CSS classes aren't APIs in the docs. We probably need to call out JavaScript globals as well, and perhaps anything in Redux.